### PR TITLE
Add undocumented member glob.has_magic

### PR DESCRIPTION
**Pull request by chadrik**
This is a handy little undocumented function that we're using in our code.

### CODE REVIEW

File: stdlib/3/glob.pyi [hunk 1]
```diff
@@ -19,3 +19,5 @@ else:
     def iglob(pathname: AnyStr) -> Iterator[AnyStr]: ...
 
 def escape(pathname: AnyStr) -> AnyStr: ...
+
+def has_magic(s: str) -> bool: ...  # undocumented
```
Comment (JelleZijlstra) [inline stdlib/3/glob.pyi]: Seems to also accept bytes:
```
def has_magic(s):
    if isinstance(s, bytes):
        match = magic_check_bytes.search(s)
    else:
        match = magic_check.search(s)
    return match is not None
```

Comment (chadrik) [reply stdlib/3/glob.pyi]: Indeed.  Thanks for catching that.

**Comment by chadrik** [comment]
I fixed the travis error, though it looks like there's an unrelated error with args to `mypy.main`.  I _think_ this is ready to merge.

**Comment by srittau** [comment]
I restarted CI.

**Pull request merged**